properties.ini

```diff
@@ -0,0 +1,157 @@
+; this file is used to define all necessary options for virt-who testing
+; all default values are null, need to update according to the test requirements
+; hypervisor-builder can help to get some hypervisor and guest information
+
+[virtwho]
+; host= is the rhel system to run virt-who service
+; repo= is the base url of the rhel compose
+host=
+username=
+password=
+port=
+repo=
+proxy_server=
+proxy_prot=
+
+[rhsm]
+server=
+username=
+password=
+prefix=
+port=
+default_org=
+
+[satellite]
+server=
+username=
+password=
+prefix=
+port=
+default_org=
+secondary_org=
+activation_key=
+
+[vcenter]
+; esx_ datas are for the esxi host which the guest is based on
+server=
+username=
+password=
+ssh_username=
+ssh_password=
+esx_ip=
+esx_username=
+esx_password=
+esx_uuid=
+esx_hwuuid=
+esx_hostname=
+esx_version=
+esx_state=
+esx_cpu=
+esx_cluster=
+guest_name=
+guest_ip=
+guest_username=
+guest_password=
+guest_uuid=
+guest_state=
+
+[xen]
+server=
+username=
+password=
+uuid=
+hostname=
+version=
+state=
+cpu=
+guest_name=
+guest_ip=
+guest_username=
+guest_password=
+guest_uuid=
+guest_state=
+
+[hyperv]
+server=
+username=
+password=
+uuid=
+hostname=
+version=
+state=
+cpu=
+guest_name=
+guest_ip=
+guest_username=
+guest_password=
+guest_uuid=
+guest_state=
+
+[rhevm]
+; vdsm_ datas are for the vdsm host which the guest is based on
+server=
+username=
+password=
+ssh_username=
+ssh_password=
+vdsm_ip=
+vdsm_username=
+vdsm_password=
+vdsm_uuid=
+vdsm_hwuuid=
+vdsm_hostname=
+vdsm_version=
+vdsm_state=
+vdsm_cpu=
+vdsm_cluster=
+vdsm_datacenter=
+guest_name=
+guest_ip=
+guest_username=
+guest_password=
+guest_uuid=
+guest_state=
+
+[libvirt-local]
```

Comment:
Could we rename this section to [local] ?
because local.server is more sense than libvirt-local.server ..

Reply:
done it and remvoed the 'cpu=' from local mode, which should also be useless.